content/en/docs/features/security.md

```diff
@@ -2,8 +2,26 @@
 title: Security
 date: 2020-01-11T14:09:21+09:00
 draft: false
+collapsible: false
 weight: 301
 ---
 
 ## Unikraft Security
+ 
+Unikernels traditionally have had serious security issues, but we argue
+that those have been implementation artifacts rather than fundamental
+problems. In fact, unikernels are used commercially in security-minded
+domains such as automotive because their high level of specialization means
+that they provide a small Trusted Computing Base. Having said that, past
+unikernel projects have failed to provide standard security features
```

Comment:
This seems a bit strong. Is it true for all / most unikernel projects?

Reply:
Yes, the same paper from NCC tackles this.